showcase/src/app/in-memory-data/services/in-memory-data.service.ts

```diff
@@ -0,0 +1,121 @@
+import {
+	InMemoryDbService,
+	ParsedRequestUrl,
+	RequestInfo,
+	RequestInfoUtilities,
+	ResponseOptions
+} from "angular-in-memory-web-api/interfaces";
+import { Observable } from "rxjs";
+import { Inject, Injectable } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+const _cloneDeep: Function = require("lodash/cloneDeep");
+const mockData: any = require("../../../../config/json-server/data.json");
```

Comment:
Do we plan to use `json-server` later or will we keep `angular-in-memory-web-api` ?
In this case, could we rename the folder where we store the data.json ?

Reply:
Okido, it was just to be sure 😊